pkg/minishift/network/settings_windows.go

```diff
@@ -41,10 +42,8 @@ var (
 )
 
 func isUsingDefaultSwitch() bool {
-	switchEnvName := "HYPERV_VIRTUAL_SWITCH"
-	switchEnvValue := os.Getenv(switchEnvName)
-
-	return switchEnvValue == "Default Switch" || switchEnvValue == ""
+	switchName := viper.GetString(configCmd.HypervVirtualSwitch.Name)
+	return switchName == "Default Switch" || switchName == nil
```

Comment:
the switch should be retrieved with PowerShell, and we should compare the `ID`, since the name is localized.